charts/cloudzero-agent/README.md

```diff
@@ -17,255 +17,36 @@ For the latest release, see [Releases](https://github.com/Cloudzero/cloudzero-ch
 
 ## Installation
 
-### Adding Helm Repository Information
+### Easy Install (Most teams will use this!)
 
 To use the chart or a beta version, you must add the repository to Helm. Refer to the [`helm repo`](https://helm.sh/docs/helm/helm_repo/) documentation for command details.
 
-#### 1. Add the Helm Chart Repository
+#### 1. Install the Helm Chart
 
 ```console
 helm repo add cloudzero https://cloudzero.github.io/cloudzero-charts
+helm install cloudzero cloudzero/cloudzero-agent \
+    --set apiKey=<YOUR_CLOUDZERO_API_KEY>\
+    --set clusterName=<YOUR_CLUSTER_NAME>
 ```
-> Note: If you intend to use a beta version, refer to the [beta installation document](./BETA-INSTALLATION.md) for the appropriate channel.
 
-#### 2. Add Repository Dependencies
+### Advanced Install (Usually when you have specific security requirements.)
 
-**Cert Manager**
-```console
-helm repo add cert-manager https://charts.jetstack.io    
-```
-
-**Metrics Exporter**
-```console
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-```
-
-#### 3. Update the Helm Repositories
-
-Ensure that the most recent chart versions are available:
-
-```console
-helm repo update
-```
-
-### Install Helm Chart
-
-The chart can be installed directly with Helm or any other common Kubernetes deployment tools.
-
-If installing with Helm directly, execute the following steps:
-
-1. Ensure that the most recent chart version is available:
-```console
-helm repo update
-```
-
-2. Provision a TLS certificate; by default, this chart deploys a `ValidatingWebhookConfiguration` resource, which requires a certificate in order validate requests to the webhook server. See related Kubernetes documentation [here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#configure-admission-webhooks-on-the-fly).
```

Comment:
Would leave out this entirely so we don't cause confusion, but document thoroughly in the values file to ensure we give folks the necessary customization for security teams.